src/versions.js

```diff
@@ -1,7 +1,15 @@
 const versions = {
     'react-cm-ui': {
         package: '9.15.0',
-        components: {},
+        components: {
+            inputs: {
+                durationPicker: {
+                    devLibraryVersion: '2.0.0',
+                    designLibraryVersion: '2.0.0',
```

Comment:
I am wondering if we should put a todo for this line too?

Reply:
Would you prefer an empty string and a `// TODO: Add this when it is available` comment?  I'm okay doing it.  Lemme know.

Reply:
Changed it to empty string and `// TODO` comment in ed7cd01.